theme/common.less

```diff
@@ -13,6 +13,11 @@ html, body {
     min-width: 320px;
 }
 
+* {
```

Comment:
why do we need this?

Reply:
For flexbox support with react-slick. see README at: https://github.com/akiran/react-slick